### DISCUSSION TITLE
My emoji design in terminal is change!!

### DISCUSSION THREAD

**Discussion by issam-seghir** [JanDeDobbeleer/oh-my-posh#3443] [Troubleshoot] [answered]
## Before
![image](https://user-images.githubusercontent.com/74573675/216544835-9c015ef1-d217-42eb-88ae-e6728d004759.png)

## After
![image](https://user-images.githubusercontent.com/74573675/216543333-e2dbb9fc-1277-44c4-8fb2-5b75354e63ae.png)

**Comment by JanDeDobbeleer** [comment, marked as answer]
The fonts are not something we control so you'll need to pick that up with the Windows Terminal team.
